src/components/activity-page/managed-interactive/managed-interactive.tsx

```diff
@@ -274,15 +274,23 @@ export const ManagedInteractive: React.ForwardRefExoticComponent<IProps> = forwa
     }
   }));
 
+  const setShowDeleteDataButton = () => {
+    return embeddable.type === "MwInteractive"
+             && embeddable.enable_learner_state
+             && embeddable.show_delete_data_button
+           || embeddable.type === "ManagedInteractive"
+             && embeddable?.library_interactive?.data.enable_learner_state
+             && embeddable?.library_interactive?.data.show_delete_data_button;
```

Comment:
because you are already assuming `embeddable` is not undefined in the earlier lines I thin you can remove the `?` from the end. So then it would be 
`embeddable.library_interactive?.data.enable_learner_state`

Other than this, it seems good to me.